commune_app/tests/test_chores.py

```diff
@@ -64,3 +74,11 @@ def test_new_chore(self, chore0):
         assert chore0.assign_to.first_name == "Elias"
         assert chore0.passed == CHORE_PASSED
         assert chore0.completed == CHORE_COMPLETED
+
```

Comment:
maybe you can add more tests about:
invalid input as @ofirtzabari did in:
https://github.com/redhat-beyond/commune/pull/61/files#diff-75216ff94ee3b4270b69c3c83dc52d7981440852ffb4de34f3b25b221f457590R54

Reply:
ill add tests in different pr